WebCryptoAPI/import_export/ec_importKey_failures_fixtures.js

```diff
@@ -0,0 +1,203 @@
+// Setup: define the correct behaviors that should be sought, and create
+// helper functions that generate all possible test parameters for
+// different situations.
+function getValidKeyData(algorithm) {
+    return validKeyData[algorithm.namedCurve];
+}
+
+function getBadKeyLengthData(algorithm) {
+    return badKeyLengthData[algorithm.namedCurve];
+}
+
+function getMissingJWKFieldKeyData(algorithm) {
+    // The curve doesn't affect when testing for missing JWK fields.
+    return missingJWKFieldKeyData["P-521"];
+}
+
+function getInvalidJWKKeyData(algorithm) {
```

Comment:
this function is only for the keys where the public key does not correspond to the private key? 

I would prefer a more specific name maybe?